test/e2e/density.go

```diff
@@ -190,8 +190,10 @@ var _ = Describe("Density [Skipped]", func() {
 	for _, testArg := range densityTests {
 		name := fmt.Sprintf("should allow starting %d pods per node", testArg.podsPerNode)
 		if testArg.podsPerNode == 30 {
-			name = "[Performance] " + name
+			name = "[Feature:Performance] " + name
 			framework.addonResourceConstraints = density30AddonResourceVerifier()
+		} else {
+			name = "[Skipped] " + name
```

Comment:
Well, everything's stable. Usually. The tests are with lower densities are just faster to run, which is really, really, really, really important for development. I treat it as smoke tests and I'd rather keep them. I'm OK with having different label for 100 pod density, and just not running it in any suite (but I guess it's pretty much equivalent to having a label Skipped/Manual).